dthm4kaiako/general/management/commands/sample_data.py

```diff
@@ -6,40 +6,52 @@
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.gis.geos import Point
-from allauth.account.models import EmailAddress
-from tests.users.factories import EntityFactory
-from resources.models import (
-    Language,
-    TechnologicalArea,
-    ProgressOutcome,
-    YearLevel,
-    CurriculumLearningArea,
-)
-from tests.resources.factories import (
-    ResourceFactory,
-    NZQAStandardFactory,
+# from tests.users.factories import EntityFactory
+# from resources.models import (
+#     Language,
+#     TechnologicalArea,
+#     ProgressOutcome,
+#     YearLevel,
+#     CurriculumLearningArea,
+# )
+# from tests.resources.factories import (
+#     ResourceFactory,
+#     NZQAStandardFactory,
+# )
+# Users
+from users.models import (
+    DietaryRequirement,
+    Entity,
 )
 # Events
 from events.models import (
+    DeletedEventRegistration,
+    EventRegistration,
     Location,
-    Series,
-)
-from tests.events.factories import (
-    EventFactory,
+    RegistrationForm,
+    # Series,
+    ParticipantType,
+    Event,
 )
+# from tests.events.factories import (
+#     EventFactory,
+# )
 # DTTA
-from tests.dtta.factories import (
-    NewsArticleFactory,
-    PageFactory,
-    ProjectFactory,
-    RelatedLinkFactory,
-)
+# from tests.dtta.factories import (
+#     NewsArticleFactory,
+#     PageFactory,
+#     ProjectFactory,
+#     RelatedLinkFactory,
+# )
 # POET
-from tests.poet.factories import (
-    POETFormResourceFactory,
-    POETFormSubmissionFactory,
-    POETFormProgressOutcomeGroupFactory,
-)
+# from tests.poet.factories import (
+#     POETFormResourceFactory,
+#     POETFormSubmissionFactory,
+#     POETFormProgressOutcomeGroupFactory,
+# )
```

Comment:
Please re-enable the other applications sample data generation process.

Reply:
Done. I've moved the user study data under the sample data generation process.